packages/application-components/src/components/main-pages/internals/main-page.styles.ts

```diff
@@ -12,5 +12,8 @@ export const Divider = styled.hr`
 `;
 
 export const MainPageContent = styled.div`
+  flex: 1;
+  flex-basis: 0;
   overflow: auto;
+  padding: ${customProperties.spacingL};
```

Comment:
Shouldn't this be `spacingM`? 🤔

Reply:
According to the design, `spacingL` was used. I will confirm with Filip.  Thanks.

Reply:
All other pages use `spacingM`, that's why I'm asking 🙂

Reply:
Filip just confirmed that it should `spacingM`.  I'll update it.

Reply:
Updated [a9c8b10](https://github.com/commercetools/merchant-center-application-kit/pull/2800/commits/a9c8b105886abd17ee946df1d37f4a3138a93057)